src/muiThemeable.js

```diff
@@ -1,23 +1,36 @@
 import React from 'react';
-import DefaultRawTheme from './styles/raw-themes/light-raw-theme';
-import ThemeManager from './styles/theme-manager';
+import getMuiTheme from './styles/getMuiTheme';
+
+const defaultTheme = getMuiTheme();
```

Comment:
This gets executed even when `context._muiTheme` is provided. Do you think it's better to lazily calculate it instead?

```js
let defaultTheme;
function getDefaultTheme() {
  return defaultTheme || (defaultTheme = getMuiTheme());
}
```